docs/releases/1.9.2.txt

```diff
@@ -9,4 +9,5 @@ Django 1.9.2 fixes several bugs in 1.9.1.
 Bugfixes
 ========
 
-* ...
+* Made :djadmin:`loaddata` raise a ``CommandError`` instead of showing
```

Comment:
This isn't a critical bugfix and so won't be backported to 1.9, however, it deserves a mention in the 1.10 release notes under "Backwards incompatible changes".